docs/architecture.md

```diff
@@ -21,6 +21,11 @@ This document describes the Olive components, and some implementation details. T
     - [Implementation](#implementation)
 - [System](#system)
     - [OliveSystem Class](#olivesystem-class)
+- [Data Container](#data-container)
```

Comment:
>Container [](http://example.com/codeflow?start=8&length=9)

do we need update the document about data container to data config?

Reply:
Actually not, the architecture doc is used to show the design of data container which is hidden from customers' config.